runtime/src/commonMain/kotlin/pbandk/internal/json/JsonMessageMarshaller.kt

```diff
@@ -38,6 +38,9 @@ internal class JsonMessageMarshaller(private val jsonConfig: JsonConfig) : Messa
             is BytesValue -> writeWrapperValue(message, message.value)
             // Other well-known types with special JSON encoding
             is Timestamp -> jsonValueMarshaller.writeString(Util.timestampToString(message))
+            is Struct -> jsonValueMarshaller.writeValue(message.fields, message.descriptor.fields.first().type)
+            is ListValue -> jsonValueMarshaller.writeRepeated(message.values, FieldDescriptor.Type.Message(Value.Companion))
```

Comment:
❓ Can this be `message.descriptor.fields.first().type` like above? If not, then this is fine too.

Reply:
That's what I went with at first, but doing so causes the `Required.Proto3.JsonInput.RepeatedListValue.JsonOutput` test to fail.